release_notes/2.2.3.md

```diff
@@ -0,0 +1,6 @@
+**WildFire Release Notes - Published by Splunk February 07, 2022**
+
+
+**Version 2.2.3 - Released February 07, 2022**
+
+* Added support for Python 3.9
```

Comment:
You may want to add the release note in `release_notes/unreleased.md` instead.
With this, it will automatically create a release with consistent format.

Reference: https://github.com/splunk-soar-connectors/chronicle/pull/11/files#diff-39546c13b2e65128c3c87daae7669c4eb99144207b446fc902bedacdd3e60345R2

Reply:
Example on how a new release note gets auto generated for you: https://github.com/splunk-soar-connectors/chronicle/commit/9d3986e143c4f690664468ddf280cd18b77272af

Reply:
Thanks @hsrivastava-crest for pointing out on the release update.
As you've mentioned, we want to update the note here and avoid generating duplicate note.
For that, we are not adding note to `unreleased.md`